charts/apple-recipe-backend/values.sandbox.template.yaml

```diff
@@ -2,6 +2,6 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
-  aadIdentityName: crumble
+  aadIdentityName: apple
```

Comment:
this needs to be added into flux,
apple-sbox-mi

cft equivalent here: https://github.com/hmcts/cnp-flux-config/blob/master/apps/cnp/sbox/base/crumble-identity.yaml

Reply:
https://github.com/hmcts/shared-services-flux/pull/1037